lib/RTCSession.js

```diff
@@ -711,22 +654,20 @@ module.exports = class RTCSession extends EventEmitter
 
         debug('emit "sdp"');
         this.emit('sdp', e);
-
-        const offer = new RTCSessionDescription({ type: 'offer', sdp: e.sdp });
-
+        //TODO: AH
         this._connectionPromiseQueue = this._connectionPromiseQueue
-          .then(() => this._connection.setRemoteDescription(offer))
+          .then(() => mediaConnection.setRemoteSdp('offer', e.sdp))
           .catch((error) =>
           {
             request.reply(488);
 
             this._failed('system', null, JsSIP_C.causes.WEBRTC_ERROR);
 
-            debugerror('emit "peerconnection:setremotedescriptionfailed" [error:%o]', error);
+            debugerror('emit "peerconnection:setremotesdpfailed" [error:%o]', error);
```

Comment:
Please replace this logs with `mediaconnection:XYZ`

```suggestion
            debugerror('emit "mediaconnection:setremotesdpfailed" [error:%o]', error);
```

Reply:
renamed